velox/exec/Operator.cpp

```diff
@@ -101,6 +101,38 @@ void Operator::maybeSetReclaimer() {
       Operator::MemoryReclaimer::create(operatorCtx_->driverCtx(), this));
 }
 
+void Operator::maybeSetQueryTracer() {
+  if (const auto& queryTraceConfig = operatorCtx_->driverCtx()->traceConfig()) {
+    if (queryTraceConfig->queryNodes.count(planNodeId()) != 0) {
+      const auto opTracePath = fmt::format(
+          "{}/{}/{}/data",
+          planNodeId(),
+          operatorCtx_->driverCtx()->pipelineId,
+          operatorCtx_->driverCtx()->driverId);
+      const auto traceDir =
+          operatorCtx_->driverCtx()->task->createTraceDirectory(opTracePath);
+      tracer_ = std::make_unique<TraceWriter>(
+          traceDir,
+          queryTraceConfig->queryTraceWriteBufferSize,
+          memory::MemoryManager::getInstance()->tracePool());
```

Comment:
Maybe provide a utility: traceMemoryPool as spillMemoryPool does for spill? thanks!

Reply:
Got it, fixed.